src/smart-app-banner/smart-app-banner.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+    CrossIcon,
+    Description,
+    DismissButton,
+    DismissContainer,
+    DownloadButton,
+    DownloadContainer,
+    LifeSgAppIcon,
+    ProceedContainer,
+    SmartAppBannerContainer,
+    TextContainer,
+    Title,
+} from "./smart-app-banner.styles";
+
+interface SmartAppBannerProps {
+    show: boolean;
+    link: string;
+    content: Content;
+    offset?: number | undefined;
+    className?: string | undefined;
+    onBannerDismiss: () => void;
+    onBannerPress?: () => void | undefined;
+}
+
+interface Content {
+    title: string;
+    buttonLabel: string;
+    buttonAriaLabel: string;
+    message?: string;
+}
+
+const APP_ICON =
+    "https://assets.life.gov.sg/react-design-system/img/app-icon/app-icon.png";
```

Comment:
just checking if this image will be added eventually? I see it in the `ci` folder but not in `prod`

Reply:
added - this url should be accessible now